spec/rails_helper.rb

```diff
@@ -60,6 +60,7 @@
 
   # Create a system_user at beginning of specs
   config.before(:all) do
+    FactoryBot.create(:country, name_en: DEFAULT_COUNTRY)
     FactoryBot.create(:user, :system)
   end
```

Comment:
If I'm not mistaken, @steveyken is strongly against using `before(:all)` rspec hooks

Reply:
lol, yes, and I'm the one that wrote the line below! I think this is a necessary evil as it seems pretty inefficient to create a default_country for each spec.

* Need to take care of destroying the country at the end of the rspec run. See `config.after(:all)`
* If rspec exits halfway through due to an invalid syntax or ctrl-c then it won't run the `config.after(:all)` hook at all. In this case, we need to add the `initialize_with` method to the countries factory so that it doesn't try to create a duplicate country with the same name the next time you run rspec

Reply:
done